elastic/kibana
# [Console] Apply console settings to new monaco editor

Closes https://github.com/elastic/kibana/issues/178966
Closes https://github.com/elastic/kibana/issues/176799

## Summary

This PR applies the console settings to the new Monaco editor:

- `fontSize`
- `wrapMode`
- `isAccessibilityOverlayEnabled`

It also modifies the `CodeEditor` component to accept an `accessibilityOverlayEnabled` prop.

**How to test:**
1. Create a `config/kibana.dev.yml` file (if one doesn't exist already) and add the line: `console.dev.enableMonaco: true`
2. Start Es and Kibana and navigate to Console
3. Click on Settings
4. Try changing the "Font size", "Wrap long lines", and "Accessibility overlay" settings and verify that the changes are correctly applied to the new monaco editor.
